Translations/bg.po

```diff
@@ -7132,206 +7347,209 @@ msgid ""
 #: Source/textdat.cpp:404
 msgid "Can't a fella drink in peace?"
-msgstr ""
+msgstr "Не може ли човек да пие на спокойствие?"
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:405
 msgid "The gal who brings the drinks? Oh, yeah, what a pretty lady. So nice, too."
-msgstr ""
+msgstr "Мадамата, която сервира напитките? А, да, каква красавица. С един мил глас."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:407
 msgid "Why don't that old crone do somethin' for a change. Sure, sure, she's got stuff, but you listen to me... she's unnatural. I ain't never seen her eat or drink - and you can't trust somebody who doesn't drink at least a little."
-msgstr ""
+msgstr "Защо тази дърта кукумявка не се заеме да прави нещо за разнообразие. Да, да, ясно ми е, че има всичко, но, слушай какво ще ти кажа... има нещо необикновено в нея. Никога не съм я виждал да яде или да пие, а да имаш вяра на някой, който не близва дори капка, не е шега работа."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:409
 msgid "Cain isn't what he says he is. Sure, sure, he talks a good story... some of 'em are real scary or funny... but I think he knows more than he knows he knows."
-msgstr ""
+msgstr "Кайн не е този, който твърди, че е. Добър разказвач е, безспорно... някои от историите са потресаващи, други смешни....  Мен ако питаш, знае много повече от това, което твърди, че знае."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:411
 msgid "Griswold? Good old Griswold. I love him like a brother! We fought together, you know, back when... we... Lazarus...  Lazarus... Lazarus!!!"
-msgstr ""
+msgstr "Грисволд? Добрият стар Грисволд. Обичам го като свой брат! Бихме се рамо до рамо навремето, когато.... двамата.... Лазар... Лазар... Лазар!!!"
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:413
 msgid "Hehehe, I like Pepin. He really tries, you know. Listen here, you should make sure you get to know him. Good fella like that with people always wantin' help. Hey, I guess that would be kinda like you, huh hero? I was a hero too..."
-msgstr ""
+msgstr "Ха-ха-ха, харесвам Пипин. Знаеш ли, поне наистина се старае. Да знаеш, трябва да го опознаеш по-отблизо. Добър човечец, който винаги е при онези, които се нуждаят от помощ. Ей, мисля си, че е като теб, герой, а? И аз бях герой..."
 
 #. TRANSLATORS: Neutral dialog spoken by Farnham (Gossip)
 #: Source/textdat.cpp:415
 msgid "Wirt is a kid with more problems than even me, and I know all about problems. Listen here - that kid is gotta sweet deal, but he's been there, you know? Lost a leg! Gotta walk around on a piece of wood. So sad, so sad..."
-msgstr ""
+msgstr "Върт си има повече проблеми, дори и от мен, знам за всички тях. Послушай ме, това хлапе - той беше там, нали разбираш? Загуби си крака! Сега ходи на дървен кол. Колко жалко, колко жалко..."
```

Comment:
```suggestion
msgstr "Върт си има повече проблеми, дори и от мен, и аз знам много за проблемите. Слушай, на това хлапе сега му върви добре, но то беше там, нали разбираш? Загуби си крака! Сега ходи на дървен кол. Колко жалко, колко жалко..."
```

Reply:
"Проблеми/те" is duplicated. Since the first part of the sentence - "има повече проблеми" / "has more problems" - indicates clearly, one could just do it like to maximize repeating of words.

"Върт си има повече проблеми, дори и от мен, и аз знам доста за това. Слушай, на това хлапе сега му върви добре, но то беше там, нали разбираш? Загуби си крака! Сега ходи на дървен кол. Колко жалко, колко жалко..."
for example.

IMHO

Reply:
Да твоето предложение е по-удачно!